bash-f/src/lib.rs

```diff
@@ -0,0 +1,180 @@
+#![no_std]
+#![doc = include_str!("../README.md")]
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
+    html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg"
+)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
+#![warn(missing_docs)]
+
+/// Number of 64-bit words in the state
+const STATE_WORDS: usize = 24;
```

Comment:
You need to make the constant public if you use it as part of the `bash_f` signature.